### ISSUE TITLE
Stacks.yaml location

### ISSUE THREAD

**Issue by brmo** [m-adawi/swarm-cd#89]
Maybe this isn't the place for this but shouldn't stacks.yaml be in the git repo you set in repos.yaml? That way swarm-cd reloads the git repo as it normally does, then reads that stacks.yaml file which then deploys the services in that file?

Odd that I have to update the stacks yaml file manually and not in a git repo, then restart/force update the swarm-cd service in order to pick up the changes and deploy.

**Comment by sanzoghenzo** [comment]
I believe this is already adressed in #23 and #54

**Comment by brmo** [comment]
In my opinion, it has a relation to, but not what I am thinking. Those tickets look to be addressing the request that if there is a directory with a compose file in it, then it should just deploy it (globbing).  I am saying the opposite where the stacks.yaml file is still necessary but it should be in the repository, not in the swarm-cd config root.

I do not agree that globbing would be the best option though. I would sometimes keep template/old/testing configurations in my repo, and of course I would not want them deploying, but defining them in the stacks file ensures that only the stacks I want deployed are.

Right now I have my swarm-cd installation inside of a docker volume using NFS as the backend so all my nodes can see the files wherever the container starts up on. But the hassle is having to go into that volume every time, update the file, and restart the service in order for it to take effect. When if the stacks.yaml search was not in the config folder but instead in the git repo, then this issue could be fixed quickly.

**Comment by sanzoghenzo** [comment]
Yeah sorry, issue 23 started more generally as an autodiscovery feature request, but I soon focused it into the globbing part because that was my use case.
#54 is still relevant, though; in fact, I believe it's exactly the same request as yours.
I was in the process of trying to insert the swarmcd stack into the stacks file to see if it can manage itself, but I ran into some troubles with my current toolchain, I hope to fix it soon and report back

**Comment by sanzoghenzo** [comment]
I'm closing this issue as duplicate of #54, let's continue the conversation there 😉
